app/assets/stylesheets/application.css.scss

```diff
@@ -40,11 +40,13 @@ body {
   font-size: 1.5em;
 }
 
-// .btn {
-//   background: #9ECAE1;
-//   color: white;
-//   font-weight: bold;
-// }
+a:hover {
+  color: white;
+}
+
+a {
+  color: #9ECAE1;
```

Comment:
Color literals like `#9ECAE1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#9ECAE1` should be written as `#9ecae1`